JavaScript-1/JS-Week-2/JS_homework/Student manager/script.js

```diff
@@ -0,0 +1,54 @@
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+
+    if (!studentName){
+        console.log("Cannot add an empty string to a class");
+            return;
+        } 
+
+
+      if(class07Students.includes(studentName)){
+        console.log(`Student ${studentName} is already in the class`);
+    return;
+    }
+    if (studentName.toLowerCase() === 'queen') {
+        class07Students.push(studentName);
+        return;
+    }
+
+    
+  if (class07Students.length >= 6) {
+    console.log(`Cannot add more students to class 07`);
```

Comment:
Code formatting is a bit messy here. Sometimes you use more spacing than other times, and this makes it hard to read. There is an extension in VS Code called Prettier. After installing it, you can set it to run by default each time a file is saved. You will see how nicely it changes formatting and how it will be easier to read your own code.
This article explains in detail how to set it up - https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code